.github/workflows/webdriverio-testing-library.yml

```
@@ -13,18 +13,18 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [12, 10]
+        node: [10, 12, 14, 16]
```

Node 15 is still supported until June 1
```suggestion
        node: [10, 12, 14, 15, 16]
```

---

We shouldn't support odd Node versions, only when they're `latest`

---

As far as I understand from [Node's LTS schedule](https://github.com/nodejs/Release), 15 is actually still supported right now, even though 16 (which will eventually be upgraded to an LTS) has been released since. Technically 16 is the only latest release here anyway.

---

Hey 😄 I've dropped Node 10 since it was causing problems when running the tests against the latest version of WebdriverIO, v7 dropped support for it so it was not unexpected. Node 10 is also 'end of life' since the 30th of April so I figured that would be fine.

While I was looking into that I saw Node 15 is 'end of life' on the 1st of June, would we be thinking of removing it then or is it better to keep it for a while to while people move over?